tests/tests.py

```diff
@@ -254,3 +257,36 @@ def handler(sender, username, **kwargs):
 
         self.assertTrue(self.signal_was_called)
 
+    def test_exceed_token_amount_per_user(self):
+
+        with override_settings(REST_KNOX=token_user_limit_knox):
+            reload_module(views)
+            for _ in range(10):
+                token = AuthToken.objects.create(user=self.user)
+            url = reverse('knox_login')
+            self.client.credentials(
+                HTTP_AUTHORIZATION=get_basic_auth_header(self.username, self.password)
+            )
+            response = self.client.post(url, {}, format='json')
+            reload_module(views)
+            self.assertEqual(response.status_code, 401)
+            self.assertEqual(response.data, {"error": "Maximum amount of tokens allowed per user exceeded."})
+
+    def test_does_not_exceed_on_expired_keys(self):
+
+        with override_settings(REST_KNOX=token_user_limit_knox):
+            reload_module(views)
+            for _ in range(9):
+                token = AuthToken.objects.create(user=self.user)
+            AuthToken.objects.create(user=self.user, expires=timedelta(seconds=0))
+            # now 10 keys, but only 9 valid so request should succeed.
+            url = reverse('knox_login')
+            self.client.credentials(
+                HTTP_AUTHORIZATION=get_basic_auth_header(self.username, self.password)
+            )
+            response = self.client.post(url, {}, format='json')
+            failed_response = self.client.post(url, {}, format='json')
+        self.assertEqual(response.status_code, 200)
```

Comment:
`reload_module(views)` missing after the `override_settings` block

Reply:
I have fixed this in latest commit.